core/templates/pages/moderator-page/services/moderator-page-backend-api.service.spec.ts

```diff
@@ -0,0 +1,110 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for Backend Api Service for
+ *  the Oppia moderator page.
+ */
+
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { fakeAsync, flushMicrotasks, TestBed, waitForAsync } from '@angular/core/testing';
+import { ActivityIdTypeDict, FeaturedActivityResponse, ModeratorPageBackendApiService, RecentCommitResponse, RecentFeedbackMessages } from './moderator-page-backend-api.service';
```

Comment:
We should avoid using more than 80 chars in a line wherever possible*
```suggestion
import { 
  ActivityIdTypeDict, FeaturedActivityResponse, 
  ModeratorPageBackendApiService, 
  RecentCommitResponse, RecentFeedbackMessages } 
  from './moderator-page-backend-api.service';
```

Can we do something around that^?

Reply:
done.